<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="home">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QLabel" name="welcomeLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>491</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Welcome to Super Groupinator!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="manageClassButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>260</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Manage Classes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createClassButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>300</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Class</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="manageClass">
     <attribute name="title">
      <string>Manage Classes</string>
     </attribute>
     <widget class="QLabel" name="className">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Class Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addStudentButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>500</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Student</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeStudentButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>500</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Student</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="classTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>731</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="exportClassButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Export Class</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="createGroup">
     <attribute name="title">
      <string>Create Groups</string>
     </attribute>
     <widget class="QLabel" name="generatorLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>531</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Group Generator</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="numStudentsGroup">
         <property name="text">
          <string># Students per group</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinStudentsGroups">
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>6</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="studentWidget_2" native="true">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>511</width>
        <height>381</height>
       </rect>
      </property>
      <widget class="QListWidget" name="groupings">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>491</width>
         <height>361</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="classWidget_2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>231</width>
        <height>381</height>
       </rect>
      </property>
      <widget class="QListWidget" name="studentList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>361</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="createGroupsButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>490</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Groups</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exportGroupsButton">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>490</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Export Groups</string>
      </property>
     </widget>
     <widget class="QLabel" name="className_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Class Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewTemplatesButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>View Templates</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveGroupButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>490</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Group</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPHYS_150">
   <property name="text">
    <string>PHYS 150</string>
   </property>
  </action>
  <action name="actionCS_20">
   <property name="text">
    <string>CS 120</string>
   </property>
  </action>
  <action name="actionCreate_New_Class">
   <property name="text">
    <string>Create New Class</string>
   </property>
  </action>
  <action name="actionOpen_Class_Roster">
   <property name="text">
    <string>Open Class List</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
